api/src/main/java/javax/enterprise/inject/spi/Annotated.java

```diff
@@ -94,6 +113,8 @@
     /**
      * <p>
      * Determine if the program element has an annotation of a certain annotation type.
+     *
+     * If the <tt>annotationType</tt> is repeatable, then returns true if at least one instance is available
```

Comment:
Up to CDI 1.2 this method should return false if you pass a repeatable annotation...